test/files.js

```diff
@@ -23,14 +23,14 @@ class ExpectedError extends Error {
 function checkNodeTypes (daemon, file) {
   return daemon.api.object.get(file.hash)
     .then(node => {
-      const meta = UnixFs.unmarshal(node.data)
+      const meta = UnixFs.unmarshal(node.Data)
```

Comment:
@vmx this looks like an API breaking change. Was it treated like so?

Reply:
Yes.

Reply:
https://github.com/ipld/js-ipld-dag-pb/releases/tag/v0.16.0